doc/source/whatsnew/v0.18.2.txt

```diff
@@ -422,3 +422,5 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+
+- Bug where ``pd.read_gbq()`` could throw ``ImportError: No module named discovery``  (:issue:`13454`)
```

Comment:
be more clear what is the issue here. Is it a changed dep, rename? do the install.rst need to change / docs?